dspace/config/dspace.cfg

```diff
@@ -1336,8 +1336,8 @@ webui.feed.item.author = dc.contributor.author
 websvc.opensearch.enable = false
 # context for html request URLs - change only for non-standard servlet mapping
 websvc.opensearch.uicontext = simple-search
-# context for RSS/Atom request URLs - change only for non-standard servlet mapping
-websvc.opensearch.svccontext = open-search/
+# context for RSS/Atom request URLs - change only for non-standard module mapping
+websvc.opensearch.svccontext = opensearch/search
```

Comment:
This should be noted in the documentation, please consider to edit or add a comment to the new wiki space for DSpace7 
https://wiki.duraspace.org/display/DSDOC7x/Business+Logic+Layer#BusinessLogicLayer-OpenSearchSupport

I don't think that this is anymore configurable so it should be removed from the configuration file and if needed to built the response turned in a constant. Again it is important to highlight the URL change in the doc @tdonohue  mabye in a general section about compatibility remarks?

Reply:
Right, I have removed the configuration option